corehq/apps/locations/models.py

```diff
@@ -625,12 +640,26 @@ def get_path_display(self):
         return '/'.join(self.get_ancestors(include_self=True)
                             .values_list('name', flat=True))
 
+    @classmethod
```

Comment:
I think this would be better as a separate function instead of a `classmethod` bolted onto the `SQLLocation` class

Reply:
The content (particularly the query filtering) of this method is closely related to this class, so I chose to keep it here.

Reply:
I'm with Emord on this one.  The functionality is very specific to one thing, and it's only used in one place, so it doesn't need to be part of the `SQLLocation` namespace.

Reply:
Alright, move it.